internal/aghtls/aghtls.go

```diff
@@ -17,7 +17,6 @@ func SaferCipherSuites() (safe []uint16) {
 			tls.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
 			tls.TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA,
 			tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
-			tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
```

Comment:
See https://github.com/AdguardTeam/AdGuardHome/issues/4925#issuecomment-1253976072.

Reply:
@ainar-g Made changes to support user preferred ciphers,